scripts/settings/Settings.gd

```diff
@@ -35,9 +40,11 @@ const SETTINGS_CONFIG = [
 		["rate",Setting.Type.FLOAT,50.0],
 		["mode",Setting.Type.ENUM,ApproachMode,ApproachMode.RATE_TIME]
 	]],
-	["parallax",Setting.Type.CATEGORY,[
-		["camera",Setting.Type.FLOAT,1.0],
-		["hud",Setting.Type.FLOAT,0.0]
+	["camera",Setting.Type.CATEGORY,[
+		["camera_parallax",Setting.Type.FLOAT,1.0],
+		["hud_parallax",Setting.Type.FLOAT,0.0],
+		["mode", Setting.Type.ENUM, CameraMode, CameraMode.HALF_LOCK],
```

Comment:
this setting and enum arent used?

Reply:
you added hud parallax

Reply:
this is referring to the camera mode enum